### PR TITLE
Remove double-dash before subscription ID in preflight script examples

### PR DESCRIPTION
**Pull request by ftouserkani-edb**
## What Changed?

Remove double-dash before subscription ID in preflight script examples.

## Checklist

Please check all boxes that apply (`[ ]` is unchecked, `[x]` is checked)

**Content**

- [ ] This PR adds new content
- [x] This PR changes existing content
- [ ] This PR removes existing content
- [ ] This PR is for a release, please add this tag:

### CODE REVIEW

**Review by drothery-edb — CHANGES_REQUESTED** [review]
We need the developer to review these changes. @ftouserkani-edb , you had a similar PR that I closed per my interpretation of the developer's comments (#2163). IAdded @xugy99 as a reviewer.

**Comment by ftouserkani-edb** [comment]
Hi Dee Dee,

I have several updates to the documentation that I would like to compile and share.
I would like to know what might be the most efficient way of updating the online documents??

Should I be doing the updates in GitHub action just like I have been doing or conveying  them to someone in your group and/or Dev if you can identify the DEV folks for me ( I can list them as required reviewers in Gtihub).

Could you please advise?


Thanks,


Frank Touserkani
Cloud Support Engineer
***@***.***

+1 781-325-8962
34 Crosby Drive, Suite 210, Bedford, MA 01730, USA

**Comment by drothery-edb** [comment]
@ftouserkani-edb , I think raising PRs is great if you suggesting specific changes such as this one. I can help tag the correct developer to review to ensure the changes are technically correct. If you have broader suggestions, you can raise a GitHub issue, email me, or create a Jira ticket.

**Comment by ftouserkani-edb** [comment]
Great. Thank You.

Frank

**Comment by gsong** [comment]
@drothery-edb Per <https://github.com/EnterpriseDB/docs/pull/2163#issuecomment-1012118409>, this change looks fine to me.

**Comment by gsong** [comment]
@drothery-edb As a side note, we should probably show the customer how to find `region`, `instance-type`, and `endpoint` options:

```sh
$ curl -sL https://raw.githubusercontent.com/EnterpriseDB/cloud-utilities/main/azure/biganimal-preflight-azure | bash -s

Usage: biganimal-preflight-azure [options] <target-subscription> <region>

Arguments:
    <target-subscription>    Azure subscription for BigAnimal deployment
    <region>                 Azure region for BigAnimal deployment

Options:
    -h, --help               Print this help message
    -i, --instance-type      Azure VM instance for BigAnimal cluster, e.g., e2s_v3
    -a, --high-availability  Plan for BigAnimal cluster with high availability enabled
    -e, --endpoint           Network endpoint flavor for BigAnimal cluster
    -r, --activate-region    Include region activation, if no clusters exist in region
    --onboard                Check if the user and subscription are configured appropriately

Behavior defaults to --onboard if no other options provided.

Examples:
    biganimal-preflight-azure --onboard 12412ab3d-1515-2217-96f5-0338184fcc04 eastus2
    biganimal-preflight-azure -i e2s_v3 --high-availability -e private 12412ab3d-1515-2217-96f5-0338184fcc04 eastus2

Available regions are: australiaeast brazilsouth canadacentral centralus eastus eastus2 francecentral japaneast northeurope norwayeast southcentralus southeastasia uksouth westeurope westus2 westus3
Available instance types are: e2s_v3 e4s_v3 e8s_v3 e16s_v3 e20s_v3 e32s_v3 e48s_v3 e64s_v3
Available endpoint flavors: public private
```

File: product_docs/docs/biganimal/release/getting_started/01_preparing_azure/index.mdx [hunk 1]
```diff
@@ -29,7 +29,7 @@ EDB provides a shell script called [`biganimal-preflight-azure`](https://github.
 
     ```sh
      curl -sL https://raw.githubusercontent.com/EnterpriseDB/cloud-utilities/main/azure/biganimal-preflight-azure | bash -s \
-      -- <subscription-ID> <azure-region> \
+      <subscription-ID> <azure-region> \
```
[the remark this answers is not in the document]
Comment (ftouserkani-edb) [reply product_docs/docs/biganimal/release/getting_started/01_preparing_azure/index.mdx:32]: Absolutely. Matching the script 'usage' output and aligning it with the UI is desirable.

[the remark this answers is not in the document]
Comment (gsong) [reply product_docs/docs/biganimal/release/getting_started/01_preparing_azure/index.mdx:32]: @ftouserkani-edb Do you want to go ahead and make those changes, either in this PR or as a separate PR?

**Comment by gsong** [comment]
@xugy99 @ftouserkani-edb What's the best way to ensure that docs is kept up to date as the script evolves?

**Comment by drothery-edb** [comment]
Merging these changes and created an issue to track additional changes the doc team plans to make per the suggestions on this PR: https://github.com/EnterpriseDB/docs/issues/2227

**Pull request merged**